### DISCUSSION TITLE
Dashboard for self hosted solution

### DISCUSSION THREAD

**Discussion by fbergamelli** [keygen-sh/community#83] [General]
Hello keygen !

I have one question. Is there any dashboard (like the one offered by your website) for the self-hosted solution (both CE and EE) ?

I've tried to setup the CE edition with docker following the steps proposed by your online doc but it seems that it lacks the dashboard.

thanks

federico

**Comment by ezekg** [comment]
Currently, no. Right now, a front-end portal will need to be built by you. But we're working on a brand new portal app that will also be open source. It will eventually replace the current dashboard, and in addition to an admin portal, it will also be a customer-facing portal.

More info on all that soon.

**Reply by fbergamelli** [reply]
oh, OK !

Thanks for the info !

have a nice day.

Federico

**Comment by MaxWaldorf** [comment]
I was curious to know if there were still any plans for a UI?

**Reply by ezekg** [reply]
Still in progress. No public ETA yet.

**Comment by fcarne** [comment]
Hello, I was wondering if there are any updates on the development of the KeyGen Portal? Thank you!

**Comment by pat-s** [comment]
A new, nextjs-based Keygen-ui attempt: [ricochet/keygen-ui](https://codefloe.com/ricochet/keygen-ui)
